inference-engine/thirdparty/clDNN/tests/test_cases/space_to_batch_gpu_test.cpp

```diff
@@ -263,6 +263,59 @@ TEST(space_to_batch_fp16_gpu, i134121_bs142121_pb010100_pe000000) {
     }
 }
 
+TEST(space_to_batch_fp16_gpu, i11611_bs1222_pb0010_pe0001_b_fs_yx_fsv16) {
```

Comment:
Please add some test cases where feature size is not multiple of 16 for b_fs_yx_fsv16 format